modules/codegen/src/smithy4s/codegen/SmithyToIR.scala

```diff
@@ -576,6 +576,24 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
       }
   }
 
+  private def maybeDefault(shape: MemberShape): List[Hint.Default] = {
+    val maybeTrait = shape.getTrait(classOf[DefaultTrait])
+    if (maybeTrait.isPresent()) {
```

Comment:
can we use `.toScala match { // regular pattern match }}` instead of the `isPresent/get`?

Reply:
@daddykotex I'm afraid not, because Scala 2.12's collection-compat does not contain OptionConverters (yes, it's annoying)

Reply:
contributions welcome :trollface: